crates/red_knot_python_semantic/src/module_resolver/resolver.rs

```diff
@@ -228,9 +228,10 @@ impl SearchPaths {
                 //  than the one resolved in the program settings because it indicates
                 //  that the `target-version` is incorrectly configured or that the
                 //  venv is out of date.
-                VirtualEnvironment::new(sys_prefix, system)
+                VirtualEnvironment::new(sys_prefix.inner.clone(), sys_prefix.origin, system)
```

Comment:
I think you can just use a reference here rather than a clone -- it should be slightly more efficient:

```suggestion
                VirtualEnvironment::new(&sys_prefix.inner, sys_prefix.origin, system)
```